services/connector/api.go

```diff
@@ -104,12 +104,16 @@ func (api *API) CallRPC(ctx context.Context, inputJSON string) (interface{}, err
 		return nil, fmt.Errorf("method %s is not allowed", request.Method)
 	}
 
-	return api.forwardRPC(ctx, request.URL, request)
+	return api.forwardRPC(ctx, request)
 }
 
 func (api *API) RecallDAppPermission(origin string) error {
```

Comment:
Let's mark this one as deprecated and make it a pure wrapper of `RecallDAppPermissionV2`